docs/enhancements/template.md

```diff
@@ -26,6 +26,16 @@ To get started with this template:
 The `Metadata` section above is intended to support the creation of tooling
 around the enhancement process.
 
+Make sure that the enhancement covers the following scenarios:
```

Comment:
I would add "... in case they are valid scenarios for the enhancement" (or another wording for it)
Because sometimes an enhancement isn't relevant to kube-api / rest-api (as an example)

Reply:
Sure